src/scales/scale.timeseries.js

```diff
@@ -87,18 +87,7 @@ class TimeSeriesScale extends TimeScale {
 
 		items.push(max);
 
-		for (i = 0, ilen = items.length; i < ilen; ++i) {
```

Comment:
Did you actually verify that memory is saved by building an table with all the timestamps vs a table with index/timestamp pairs that break the linearity (in typical use case weekends and dst are the breaking points)

Reply:
I tried, but the memory numbers in the Chrome developer tools performance tab were very volatile. I couldn't detect any difference. I'm not sure if there's a better way to measure this. However, I think that probably the more important aspect here is that by having every point in the table we can do a lookup by index instead of a binary search